include/drmpp/drm/composition.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2024 drmpp contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef INCLUDE_DRMPP_DRM_COMPOSITION_H_
+#define INCLUDE_DRMPP_DRM_COMPOSITION_H_
+
+#include <memory>
+#include <span>
```

Comment:
This is only allowed >= C++ 20.  This project is based on C++ 17.

Reply:
Ah, true. I used span at the beginning but forgot to remove this apparently.